### DISCUSSION TITLE
Installation ERROR

### DISCUSSION THREAD

**Discussion by himanshuahuja96** [directus/directus#13536] [Archive]
I am trying to install Directus with [PlanetScale DB](https://planetscale.com), but I'm not able to complete the installation.

? Choose your database client MySQL / MariaDB / Aurora
? Database Host: ***********.ap-south-2.psdb.cloud
? Port: 3306
? Database Name: directus
? Database User: ************
? Database Password: *****************************************************

### To Reproduce

1) npm init directus-project directus_with_planetscale
2) And select MySQL
3) Goto [Planetscale](https://planetscale.com) Create an Account and Database
4) After the database is created, Create a new branch (whatever name you like. I created with the name of DEV)
5) Click "Connect" and you will see connection related information
6) Paste the connection string as directus cli is directing.

### Errors Shown

```
Something went wrong while seeding the database:

[ER_UNKNOWN_ERROR] ER_UNKNOWN_ERROR: unknown error: Code: UNAVAILABLE
server does not allow insecure connections, client must use SSL/TLS
```

**Comment by azrikahar** [comment]
Disclaimer: I don't personally have in depth experience with PlanetScale, so I could be wrong.

Currently PlanetScale is not officially supported _yet_, particularly due to not supporting [foreign key constraints](https://docs.planetscale.com/learn/operating-without-foreign-key-constraints) (although it is said to be MySQL compatible, but not in this context at the moment).

As for the SSL issue, you should be able to resolve it by modifying your env variables as described here: https://github.com/directus/directus/discussions/12527#discussioncomment-2489842, but once again resolving this particular SSL issue will not be the primary hurdle in this scenario.

**Reply by himanshuahuja96** [reply]
That is correct, foreign key is also not supported. Is there a way we can say don't use database foreign key constraints? There are some ORM that works without foreign key constraints. For example, Prisma is one of them.

**Reply by azrikahar** [reply]
Currently there isn't a way to do that, but support for more database vendors is always something we'd love! Granted it could be quite non-trivial to have full support without foreign key constraints since there is a reliance on it for things such as cascade delete, but there is a recent discussion for Vitess support (powering PlanetScale) in #13395 so perhaps we can see what are the community or other core members' thoughts on it.
